CHANGES.rst

```diff
@@ -6,6 +6,9 @@ dev (master)
 
 * ... [Short description of non-trivial change.] (Issue #)
 
+* Made the connection pool retry on ``SSLError``.  The original ``SSLError``
+  is available on ``MaxRetryError.reason``. (Issue #1112)
+
```

Comment:
This has ended up in the wrong place: it should be above the placeholder, not below.